src/components/mortgage-investment-compare/input-card.js

```diff
@@ -250,11 +272,16 @@ const InputCard = props => {
         </FormGroup>
         <hr />
         <FormGroup className='mb-3'>
-          <label>Inflation</label>
+          <Tooltip
+            children={<label>Inflation</label>}
+            icon={faQuestionCircle}
+            content='How fast you expect prices to rise every year'
+            id='inflation'
+          />
           <InputGroup className='my-2'>
             <ValidatedNumberInput
-              max={200}
-              min={-200}
+              max={1000000000}
+              min={-1000000000}
```

Comment:
yes, this is possible

Reply:
An issue I ran into is when you max out all the numbers, then the app crashes. It doesn't happen when you max out an individual field

Reply:
maybe 1.7976931348623157e+308? lol I'll give it a try later